scripts/menuHelper.lua

```diff
@@ -126,16 +224,25 @@ function menuHelper.effects.runGlobalFunction(inputObjectName, inputFunctionName
 end
 
 -- Deprecated
-function menuHelper.effects.setDataVariable(inputVariable, inputValue)
-    return menuHelper.effects.setPlayerDataVariable(inputVariable, inputValue)
+function menuHelperEffects.setDataVariable(inputVariable, inputValue)
+    return menuHelperEffects.setPlayerDataVariable(inputVariable, inputValue)
 end
 
 -- Deprecated
-function menuHelper.effects.runFunction(inputFunctionName, inputArguments)
-    return menuHelper.effects.runPlayerFunction(inputFunctionName, inputArguments)
+function menuHelperEffects.runFunction(inputFunctionName, inputArguments)
+    return menuHelperEffects.runPlayerFunction(inputFunctionName, inputArguments)
 end
```

Comment:
Please don't change the code if you don't have to. 

You can add a `---@deprecated` flag to this function, and the one above.